pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -116,6 +118,22 @@ func LoopDevicesAllowed() bool {
 	return loopDevicesAllowed
 }
 
+func SetEnforceHostNetwork(data map[string]string) {
+
+	val := k8sutil.GetValue(data, EnforceHostNetworkSettingName, EnforceHostNetworkDefaultValue)
+	ret, err := strconv.ParseBool(val)
+
+	if err != nil {
```

Comment:
By Go convention, the `if err != nil` should be paired with the previous call and not separated by a newline.

Reply:
done